website/docs/d/linux_function_app.html.markdown

```diff
@@ -287,7 +287,7 @@ A `custom_oidc_v2` block supports the following:
 
 * `client_id` - The ID of the Client to use to authenticate with the Custom OIDC.
 
-* `openid_configuration_endpoint` - The app setting name that contains the `client_secret` value used for the Custom OIDC Login.
+* (Required) Specifies the endpoint used for OpenID Connect Discovery. For example `https://example.com/.well-known/openid-configuration`.
```

Comment:
We seem to be missing the property name here, additionally since these are the data source docs, we don't include whether a property is `Optional` or `Required` unless it's an input for the data source which most aren't 
```suggestion
* `openid_configuration_endpoint`- The endpoint used for OpenID Connect Discovery. For example `https://example.com/.well-known/openid-configuration`.
```